stable/pomerium/Chart.yaml

```diff
@@ -1,10 +1,11 @@
 apiVersion: v1
 name: pomerium
-version: 4.2.0
+deprecated: true
+version: 4.2.1
 appVersion: 0.5.2
 home: http://www.pomerium.io/
 icon: https://www.pomerium.io/logo.svg
-description: Pomerium is an identity-aware access proxy.
+description: DEPRECATED Pomerium is an identity-aware access proxy.
```

Comment:
@travisgroth Maybe point to the new repo on the description line since that's what people will see when they do a repo search?

Reply:
👍